tools/build_pytorch_libs.bat

```diff
@@ -140,11 +147,11 @@ FOR %%a IN (%_BUILD_ARGS%) DO (
 : Copy Artifacts
 cd torch\lib
 
-copy /Y tmp_install\lib\* .
-IF EXIST ".\tmp_install\bin" (
-  copy /Y tmp_install\bin\* .
+copy /Y "%CMAKE_INSTALL_PREFIX%\lib\*" .
```

Comment:
You could just make this `copy /Y "%INSTALL_DIR%\lib\*" .`, etc. if we got rid of CMAKE_INSTALL_PREFIX, right?